src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +58,10 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getSex() == Person.Sex.MAN
```

Comment:
do not use abbreviated words
```suggestion
                .filter(person -> person.getSex() == Person.Sex.MAN
```